search-service/src/test/kotlin/com/egm/stellio/search/authorization/EnabledAuthorizationServiceTests.kt

```diff
@@ -177,21 +179,21 @@ class EnabledAuthorizationServiceTests {
                 eq(Some(subjectUuid)),
                 eq(entityId01),
                 emptyList(),
-                listOf(AccessRight.R_CAN_ADMIN)
+                listOf(R_IS_OWNER, R_CAN_ADMIN)
             )
         }
     }
 
     @Test
     fun `it should create admin link for a set of entities`() = runTest {
```

Comment:
the name of the method should be renamed too  :  ` it should create creator link for a set of entities`

Reply:
and even `it should create owner link ...`